### ISSUE TITLE
Is methamphetamine in the results of the A.3 query?

### ISSUE THREAD

**Issue by vgardner-renci** [NCATSTranslator/minihackathons#238]
https://arax.ncats.io/?source=ARS&id=b018e158-2139-4a6b-bbe0-28d22110b8c4

**Comment by sstemann** [comment]
@vdancik just checking in on this one - the notebook is here:  https://github.com/NCATSTranslator/minihackathons/blob/main/2021-12_demo/workflowA/KCNMA1DemoNotebook.ipynb

When @MarkDWilliams submitted the notebook last month we did great results.

**Comment by vdancik** [comment]
@sstemann It appears that none of the MolePro sources have a connection between KCNMA1 and methamphetamine.

**Comment by vgardner-renci** [comment]
@vdancik Is it out of scope for Molepro?

**Comment by vdancik** [comment]
No, it is not out of scope. MolePro has connections between methamphetamine and other genes, but not KCNMA1.
